### DISCUSSION TITLE
Plans of moving SCB project to 'Flagship' category?

### DISCUSSION THREAD

**Discussion by sourabhsdeshpande** [secureCodeBox/secureCodeBox#1108] [General]
Hi,

Found that it has moved from 'Incubator' to 'Lab' category recently, which is great!

Is there a plan to move SCB as a 'Flagship' project in near future?
If yes, what is the timeline?

**Comment by J12934** [comment]
Hi 👋

yes totally, we'd love to move up to the Flagship status.

Its hard to give any timeline for when that's gonna happen thought as the decision to move up in the OWASP maturity categories is not decided by us 😀
At least from my observations, projects don't usually get promoted more then once a year, so I would not expect it in the next months.
